core/server/helpers/author.js

```diff
@@ -10,32 +10,30 @@
 // Block helper: `{{#author}}{{/author}}`
 // This is the default handlebars behaviour of dropping into the author object scope
 
-var proxy = require('./proxy'),
-    _ = require('lodash'),
-    SafeString = proxy.SafeString,
-    handlebars = proxy.hbs.handlebars,
-    templates = proxy.templates,
-    url = proxy.url;
+const {SafeString, templates, url, hbs: {handlebars}} = require('./proxy'),
+    isString = require('lodash/isString'),
+    ESCAPE = require('lodash/escape'); // must be capital to avoid naming collision
```

Comment:
Due to scoping, there shouldn't be any collisions - kind of like how if you want to use bluebird for a specific file (generally, not in Ghost because of overriding) you can just do `const Promise = require('bluebird')`. Since the global escape function is not used here, there's no need to try and avoid collisions.

Reply:
is anyone using it?

Reply:
Each file has its own scope, so as long as it's not used in this file there's nothing to worry about 😄